### PR TITLE
feature/FOUR-18976

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
FOUR-18972 : Implement the new breadcrumbs

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-18976

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-83b0a338e2.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-83b0a338e2.engk8s.processmaker.net

File: resources/jscomposition/cases/casesMain/CasesMain.vue [hunk 1]
```diff
@@ -30,24 +31,35 @@ export default defineComponent({
...
+      pages.value.pop();
+      pages.value.push({ name: counter.header, current: true });
+
       router.push({ path: counter.url }).catch((e) => {});
     };
 
     onMounted(async () => {
       const resCounters = await getCounters();
       countersData.value = formatCounters(resCounters);
+      const currentCounter = countersData.value.find((counter) => counter.url === route.path);
```
Comment (henryjonathanquispe) [inline resources/jscomposition/cases/casesMain/CasesMain.vue:50]: Subir esta linea
   La declaracion de const en una sola seccion

**Review by henryjonathanquispe — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-83b0a338e2.engk8s.processmaker.net

**Pull request merged**